### DISCUSSION TITLE
Version warning hard to read in dark mode

### DISCUSSION THREAD

**Discussion by patrick-5546** [squidfunk/mkdocs-material#5062] [Q&A] [answered]
I copied the version warning html from the documentation: https://squidfunk.github.io/mkdocs-material/setup/setting-up-versioning/?h=ver#version-warning. However, there is not enough contrast between the version warning background and link color in dark mode. Additionally, the background clashes with the header color. Would this be easy to fix?

<img width="390" alt="image" src="https://user-images.githubusercontent.com/65756895/219543491-0477ac98-fa0c-45ce-ad50-134ed88b5b70.png">

**Comment by squidfunk** [comment]
You're right – dark mode doesn't currently work particularly well with the version warning. Could I please ask you to create a new issue with a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/)? It's likely an easy fix then.

**Comment by squidfunk** [comment, marked as answer]
Fixed in 981ee6549. The version warning will now always use the light scheme, so it's consistent:

![screenshot-localhost-8000-mkdocs-material-getting-started-1676716632790](https://user-images.githubusercontent.com/932156/219855933-797a2a24-9dad-4a14-807d-32f1269fc943.png)

Additionally, two new CSS variables were introduced that allow for overriding the background and foreground color:

https://github.com/squidfunk/mkdocs-material/blob/981ee65498a15c35f572651636f2272fb3dfc399/src/assets/stylesheets/main/_colors.scss#L114-L116
